clippy_lints/src/utils/internal_lints.rs

```diff
@@ -395,3 +402,181 @@ fn is_trigger_fn(fn_kind: FnKind<'_>) -> bool {
         FnKind::Closure(..) => false,
     }
 }
+
+declare_lint_pass!(CollapsibleCalls => [COLLAPSIBLE_SPAN_LINT_CALLS]);
+
+impl  EarlyLintPass for CollapsibleCalls {
+    fn check_expr(&mut self, cx: &EarlyContext<'_>, expr: &AstExpr) {
+        use ast::{StmtKind, ExprKind};
+
+        span_lint_and_help(cx, COLLAPSIBLE_SPAN_LINT_CALLS, expr.span, "lint message", "help message");
```

Comment:
I am trying to address the issues, when I come amost close, I found the span_lint_and_help doesn't even emit in stdout. I have try `utils/dev update_lints` but nothing changed?

Reply:
Have you registered the lint pass?

Reply:
I have tried to search other internal lints like `PRODUCE_ICE` on the repo, and do the same thing for internal lint I am writing, but it didn't work. 

Perhaps because I didn't search `ProduceIce` either. I'll tried that out.

Reply:
`update_lints` currently doesn't update internal lints. See #5156

In addition to that, you have to add `store.register_early_pass(..)` to the lib file by hand. See https://github.com/rust-lang/rust-clippy/blob/master/doc/adding_lints.md#lint-declaration